### DISCUSSION TITLE
Secrets variables not updating at all

### DISCUSSION THREAD

**Discussion by Garenv** [community/community#124588] [Actions]
### Select Topic Area

Bug

### Body

I'm trying to update my the value of Secrets to store environment variables but it doesn't save.  I've tried clicking out of the textarea after populating said environment variables and I've also tried on a different browser other than Chrome but I'm getting the same results.

Anyone else encountering this as well?  Is there any way around it?  I need this to work so that my CI/CD pipeline can be operational.  I've attached a screenshot for reference.

<img width="960" alt="Screenshot 2024-05-18 at 11 22 16 PM" src="https://github.com/community/community/assets/22897386/f83ceb49-0867-4bfa-a895-58ea95811194">

**Comment by ASaiAnudeep** [comment]
+1 facing the same issue.

**Comment by hasseebmayo** [comment]
+1 facing the same issue.
